packages/plugins/rubygem-smart_proxy_dhcp_dnsmasq/rubygem-smart_proxy_dhcp_dnsmasq.spec

```diff
@@ -1,31 +1,40 @@
 # template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_datadir:%global _root_datadir %{_datadir}}
+%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
 %global gem_name smart_proxy_dhcp_dnsmasq
 %global plugin_name dhcp_dnsmasq
 
-%global foreman_proxy_min_version 1.15.0
-%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir %{_root_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_root_localstatedir}/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
-%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+%global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
+%global smart_proxy_dynflow_bundlerd_dir %{!?scl:/opt/theforeman/tfm/root}%{_datadir}/smart_proxy_dynflow_core/bundler.d
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.7
-Release: 1%{?foremandist}%{?dist}
+Release: 3%{?foremandist}%{?dist}
 Summary: dnsmasq DHCP provider plugin for Foreman's smart proxy
 Group: Applications/Internet
-License: GPLv3
+License: GPL-3.0
```

Comment:
```suggestion
License: GPLv3
```

Reply:
Arg. This license difference is annoying.

Reply:
Probably easiest to add a translation table in the template: https://github.com/theforeman/foreman-packaging/pull/4180 is incomplete, but the mechanism is easily extendable.